src/pepr/policies/exemptions/index.ts

```diff
@@ -1,52 +1,39 @@
 import { KubernetesObject } from "kubernetes-fluent-client";
 import { Log, PeprMutateRequest, PeprValidateRequest } from "pepr";
-
-export type Exempt = {
-  /**
-   * Namespace of the resource to exempt.
-   */
-  namespace?: string;
-  /**
-   * Name of the resource to exempt. Can be a regular expression.
-   */
-  name?: string | RegExp;
-};
-
-export type ExemptList = Array<Exempt>;
+import { Policy } from "../../operator/crd";
+import { Store } from "../common";
 
 /**
  * Register a list of exemptions to be used by the validation action.
```

Comment:
```suggestion
 * Check a resource against an exemption list for use by the validation action.
```
Something like this to reword I think?